contracts/interfaces/IQuoterV3.sol

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.7.5;
+pragma abicoder v2;
+
+/// @title QuoterV2 Interface
+/// @notice Supports quoting the calculated amounts from exact input or exact output swaps.
+/// @notice For each pool also tells you the number of initialized ticks crossed and the sqrt price of the pool after the swap.
+/// @dev These functions are not marked view because they rely on calling non-view functions and reverting
+/// to compute the result. They are also not gas efficient and should not be called on-chain.
+interface IQuoterV3 {
```

Comment:
Curious - is there a reason why only some functions are in the interface?

Reply:
don't need to include the private functions. they are internal/ helpers and aren't necessarily required functionality of any general quoter that inherits from this interface

Reply:
Ah I see thanks